test/db/index.js

```diff
@@ -565,6 +565,35 @@ describe('db', function() {
     });
   });
 
+
+  describe('scopes', function () {
+
+    it('can register and fetch scopes', () => {
+      const scopeName = 'https://some-scope.mozilla.org';
+      const newScope = {
+        scope: scopeName,
+        hasScopedKeys: true
+      };
+      return db.registerScope(newScope)
+        .then(
+          function() {
+            return db.getScope(scopeName);
```

Comment:
What should this do if the named scope does not exist?  Whatever the answer, we should probably have an explicit test for it.

Reply:
Yeah, same as .getClient it resolves with `undefined`